ui/pages/api/cabinSorting/sortedHackers.ts

```diff
@@ -0,0 +1,125 @@
+import { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
There's a lot going on in this file. Is there a way we can modularize it into separate files?

Reply:
Or is this how it is typically organized next?